packages/dx-react-grid/src/plugins/grouping-state.jsx

```diff
@@ -65,6 +80,27 @@ export class GroupingState extends React.PureComponent {
             grouping,
           ]}
         />
+        <Watcher
```

Comment:
Why do you use Watcher to track changes in local Plugin context? It seems like overkill.